### PR TITLE
add is_command_callable functionality

### LINKED ISSUE #31: Loglevel bounding logic yields always-silent behavior
Noticed by @nsheff https://github.com/databio/pararead/commit/158964c02ec79f809b254f002944dd731d2a5f25

### PR DESCRIPTION
**Pull request by nsheff**
In addition to new functionality, close #31

### CODE REVIEW

File: pararead/logs.py [hunk 1]
```diff
@@ -332,7 +332,7 @@ def _level_from_verbosity(verbosity):
     elif isinstance(verbosity, int):
         # Allow negative value to mute even ERROR level but not CRITICAL.
         # Also handle excessively high verbosity request.
-        v = min(min(verbosity, 0), len(LEVEL_BY_VERBOSITY) - 1)
+        v = min(max(verbosity, 0), len(LEVEL_BY_VERBOSITY) - 1)
```
Comment (vreuter) [inline pararead/logs.py]: This change invalidates the comments above it.

**Comment by nsheff** [comment]
what is this stale PR doing here sitting around for 3 months... I don't even remember this...

File: pararead/logs.py (hunk 1 above)
Comment (nsheff) [reply pararead/logs.py]: go ahead and change...

Comment (vreuter) [reply pararead/logs.py]: I don't know what the change is trying to do, though; what's the intended behavior?

Comment (vreuter) [reply pararead/logs.py]: Is it that we no longer want the comments to be true, or that they should be?

Comment (nsheff) [reply pararead/logs.py]: I don't think I made that change, or at least I don't recall it. TO be honest I have no idea what's going on with this logger stuff, still, or what these changes would do... It's not immediately clear to me and I would need some time to try to figure out how your logging system works... I just use it (and it's awesome)

**Pull request closed without merging**